rustls/examples/internal/bench_impl.rs

```diff
@@ -131,6 +250,15 @@ enum ResumptionParam {
 }
 
 impl ResumptionParam {
+    fn from_subcommand(cmd: &Command) -> Self {
```

Comment:
Nit: if we're going to use `Command` here, maybe it should go below `ResumptionParam`? (This is admittedly one of those cases where a strict ordering might not make sense.)

Reply:
I think `Command` warrants being pretty high up in this file given its value in explaining what comes after it.

Reply:
Fair!